sdk/cosmos/azure-cosmos-spark_3_2-12/Samples/DatabricksLiveContainerMigration/CosmosDBLiveContainerMigrationValidation.scala

```diff
@@ -1,56 +1,105 @@
+    
+    return spark.sql(query)
+}
+
+def checkCosmosSourceTargetCountDiff(cosmosViewDatabaseName: String, cosmosSourceViewName: String, cosmosTargetViewName: String): DataFrame = {
+    val query = s"""-- anti join shows all documents(versions) in the SourceView not present in SinkView
+                        SELECT COUNT(*) FROM cosmosCatalog.`$cosmosViewDatabaseName`.`$cosmosSourceViewName` source
+                        LEFT ANTI JOIN cosmosCatalog.`$cosmosViewDatabaseName`.`$cosmosTargetViewName` target
+                        ON source.id = target.id and source._etag == target._etag
+                """
+    
+    return spark.sql(query)
+}
 
 // COMMAND ----------
 
-// MAGIC %sql
-// MAGIC CREATE TABLE IF NOT EXISTS cosmosCatalog.`database-v4`.SourceView 
-// MAGIC   (id STRING, _etag STRING)
-// MAGIC USING cosmos.oltp
-// MAGIC TBLPROPERTIES(isCosmosView = 'True')
-// MAGIC OPTIONS (
-// MAGIC   spark.cosmos.database = 'database-v4', -- change database-v4 to be the value of your source database 
-// MAGIC   spark.cosmos.container = 'customer', -- change customer to be the value of your source container 
-// MAGIC   spark.cosmos.read.inferSchema.enabled = 'False',  
-// MAGIC   spark.cosmos.read.partitioning.strategy = 'Default');
-// MAGIC 
-// MAGIC SELECT * FROM cosmosCatalog.`database-v4`.SourceView
+val cosmosSourceViewName = cosmosSourceContainerName + '_' + randomUUID
+val cosmosTargetViewName = cosmosTargetContainerName + '_' + randomUUID
+createCosmosView(cosmosDatabaseName = cosmosSourceDatabaseName, cosmosContainerName = cosmosSourceContainerName, cosmosViewDatabaseName = cosmosSourceContainerName, cosmosViewName = cosmosSourceViewName)
+createCosmosView(cosmosDatabaseName = cosmosTargetDatabaseName, cosmosContainerName = cosmosTargetContainerName, cosmosViewDatabaseName = cosmosSourceContainerName, cosmosViewName = cosmosTargetViewName)
 
 // COMMAND ----------
 
-// MAGIC %sql
-// MAGIC CREATE TABLE IF NOT EXISTS cosmosCatalog.`database-v4`.SinkView 
-// MAGIC   (id STRING, _origin_etag STRING)
-// MAGIC USING cosmos.oltp
-// MAGIC TBLPROPERTIES(isCosmosView = 'True')
-// MAGIC OPTIONS (
-// MAGIC   spark.cosmos.database = 'database-v4', -- change database-v4 to be the value of your target database 
-// MAGIC   spark.cosmos.container = 'customer_v2', -- change customer_v2 to be the value of your target container
-// MAGIC   spark.cosmos.read.inferSchema.enabled = 'False',  
-// MAGIC   spark.cosmos.read.partitioning.strategy = 'Default');
-// MAGIC   
-// MAGIC SELECT * FROM cosmosCatalog.`database-v4`.SinkView
+val sourceTargetDiff = checkCosmosSourceTargetDiff(cosmosViewDatabaseName = cosmosSourceContainerName, cosmosSourceViewName = cosmosSourceViewName, cosmosTargetViewName = cosmosTargetViewName)
+val sourceTargetCountDiff = checkCosmosSourceTargetCountDiff(cosmosViewDatabaseName = cosmosSourceContainerName, cosmosSourceViewName = cosmosSourceViewName, cosmosTargetViewName = cosmosTargetViewName)
```

Comment:
When running validation I got:
[UC_NOT_ENABLED] Unity Catalog is not enabled on this cluster
Can you please include instructions in README.md for how users that are unfamiliar Databricks (this will be a good number as they might only be using this approach for container migration) can avoid this?